service/src/test/java/com/ericsson/eiffel/remrem/generate/service/EiffelRemremControllerUnitTest.java

```diff
@@ -48,56 +50,57 @@ public void setUp() throws Exception {
         Mockito.when(service.getServiceName()).thenReturn("eiffelsemantics");
         Mockito.when(service2.getServiceName()).thenReturn("eiffel3");
         
+        String successOutput = "{\"meta\":{\"id\":\"646cb3ff-30b5-42f7-b686-220131287b2d\",\"type\":\"EiffelActivityFinishedEvent\",\"version\":\"1.0.0\",\"time\":1487851528091,\"tags\":[\"tag1\",\"tag2\"],\"source\":{\"domainId\":\"domainID\",\"host\":\"host\",\"name\":\"name\",\"serializer\":{\"groupId\":\"G\",\"artifactId\":\"A\",\"version\":\"V\"},\"uri\":\"http://java.sun.com/j2se/1.3/\"}},\"data\":{\"outcome\":{\"conclusion\":\"TIMED_OUT\",\"description\":\"Compilation timed out.\"},\"persistentLogs\":[{\"name\":\"firstLog\",\"uri\":\"http://myHost.com/firstLog\"},{\"name\":\"otherLog\",\"uri\":\"http://myHost.com/firstLog33\"}],\"customData\":[]},\"links\":[{\"type\":\"LinkTargetType\",\"target\":\"LinkTarget\"}]}";
+        String failureOutput = "{\"message\":\"Unknown message type requested\",\"cause\":\"\u0027EiffelActivityFinis\u0027 is not in the vocabulary of this service\"}";
+
+        
         Mockito.when(service.generateMsg(
-            Mockito.eq("eiffelactivityfinished"),
-            Mockito.anyObject()
-        )).thenReturn("{ \"result\":\"SUCCESS\" }");
+                Mockito.eq("EiffelActivityFinishedEvent"),
+                Mockito.anyObject())).thenReturn(successOutput);
         
+
         Mockito.when(service.generateMsg(
-                Mockito.eq("eiffeljobfinished"),
-                Mockito.anyObject()
-        )).thenReturn("{ \"result\":\"FAILURE\" }");
+                Mockito.eq("EiffelActivityFinished"),
+                Mockito.anyObject())).thenReturn(failureOutput);
         
         Mockito.when(service2.generateMsg(
-                Mockito.eq("eiffeljobfinished"),
-                Mockito.anyObject()
-        )).thenReturn("{ \"result\":\"SUCCESS\" }");
-        
+                Mockito.eq("eiffelartifactnew"),
+                Mockito.anyObject())).thenReturn(successOutput);
         
         Mockito.when(service2.generateMsg(
-                Mockito.eq("eiffelactivityfinished"),
-                Mockito.anyObject()
-        )).thenReturn("{ \"result\":\"FAILURE\" }");
-        
+                Mockito.eq("eiffelartifactnewevent"),
+                Mockito.anyObject())).thenReturn(failureOutput);
+     
     }
     
-    @Test 
-    public void testSemanticsEvent() throws Exception {        
-        JsonElement elem = unit.generate("eiffelsemantics", "eiffelactivityfinished", body.getAsJsonObject());
-        assertEquals(elem.getAsJsonObject().get("result").getAsString(), "SUCCESS");
+    @Test
```

Comment:
Are we testing anything? Even the outout we generated is mocked on top? Not very good with mockito, so some one else can have a look